package.json

```diff
@@ -1,20 +1,30 @@
 {
   "name": "fedordead",
+  "description": "Website for FED || Dead",
   "version": "1.0.0",
   "scripts": {
-    "scss": "node-sass --output-style compressed -o assets/css src/scss/style.scss",
-    "autoprefixer": "postcss -u autoprefixer -r assets/css/style.css",
-    "compile": "mustache posts.json index.mustache > index.html",
-    "build:css": "npm run scss && npm run autoprefixer",
-    "watch": "onchange 'src/scss/**.scss' -- npm run build:css",
-    "build": "npm run build:css && npm run compile"
+    "svg-min": "svgo -f src/_assets/img -o dist/img",
+    "compile-html": "mustache src/data/posts.json src/index.mustache > dist/index.html",
+    "compile-scss": "node-sass --output-style compressed -o dist/css src/_assets/scss/style.scss",
+    "autoprefixer": "postcss -u autoprefixer -r dist/css/*",
+    "watch:css": "onchange 'src/_assets/scss/**.scss' -- npm run build:css",
+    "watch:html": "onchange 'src/**.mustache' -- npm run compile-html",
+    "watch:all": "parallelshell 'npm run watch:css' 'npm run watch:html'",
+    "build:css": "parallelshell 'npm run compile-scss' 'npm run autoprefixer'",
+    "build:all": "parallelshell 'npm run build:css' 'npm run compile-html' 'npm run svg-min'"
```

Comment:
`&&` doesn't work properly with the `watch` scripts, I was stuck in loops, see: https://css-tricks.com/why-npm-scripts/#article-header-id-16.